manifests/a/AnyAssociation/Anytype/0.35.2/AnyAssociation.Anytype.locale.en-US.yaml

```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: AnyAssociation.Anytype
+PackageVersion: 0.35.2
+PackageLocale: en-US
+Publisher: Any Association
+PublisherUrl: https://github.com/anyproto
+PublisherSupportUrl: https://anytype.io/faq
+PrivacyUrl: https://anytype.io/website_privacy
+Author: Any Association
+PackageName: Anytype
+PackageUrl: https://download.anytype.io/
+License: Proprietary
+Copyright: Copyright © 2023 Any Association
+ShortDescription: The everything app for those who celebrate trust & autonomy.
+Documentations:
+- DocumentLabel: Anytype Docs (English)
+  DocumentUrl: https://doc.anytype.io/anytype-docs/intro/readme
+- DocumentLabel: Anytype 文档（简体中文）

```

Comment:
If we want to add this, it should really be under a separate locale file **"AnyAssociation.Anytype.locale.`zh-CN`.yaml"** with `PackageLocale: zh-CN`. This will ensure only users of chinese locale see this Document node. You can translate other fields and add them in the new locale file as necessary (There is no restriction to add/translate all fields tho)

Reply:
I'm wondering if there're Simplified Chinese users of this package. I'd prefer remove this section, because users can switch languages on the Portal site on their own.